src/io_meandata.F90

```diff
@@ -385,7 +377,7 @@ subroutine ini_mean_io(ice, dynamics, tracers, partit, mesh)
 !_______________________________________________________________________________
 ! output surface forcing
 CASE ('fh        ')
-    call def_stream(nod2D, myDim_nod2D, 'fh',       'heat flux',                       'W/m2',      heat_flux_in(:),           io_list(i)%freq, io_list(i)%unit, io_list(i)%precision, partit, mesh)
```

Comment:
I seem to recall this was a recent fix, that seems to be made undone here. Probably a mistake.

Reply:
Strange ... I did not touch gen_modules_clock.F90 and io_meandata.F90 at these lines.

Reply:
It it probably because you based you changes upon an older version of fesom. I'd suggest you rebase your changes onto the latest main

Reply:
When I run 
esm_master install-awiesm-2.6
esm_master install-fesom-2.6
esm_master install-fesom-2.6-paleodyn
I always obtain io_meandata.F90 without these lines.

Reply:
That's because we download FESOM 2.6.1 right now via esm_tools. But here we are merging in the main branch. These lines have been additions to the main branch in the last few weeks. The strange thing for me, is that git removes them again in this PR, because you don't touch the same lines in your branch. 

We can have a look together after the holidays to sort this out.